src/test/java/com/puppycrawl/tools/checkstyle/filters/AbstractPatchFilterEvaluationTest.java

```diff
@@ -91,7 +92,31 @@ protected void testByConfig(String configPath)
             final String expectedFile = configPath.replaceFirst(
                     "(default|zero)ContextConfig.xml", "expected.txt");
             final List<String> expected = Files.readAllLines(Paths.get(getPath(expectedFile)));
-            final List<String> actuals = lnr.lines().limit(expected.size()).sorted()
+            final Comparator<String> comparator = new Comparator<String>() {
+                @Override
+                public int compare(String o1, String o2) {
+                    final int lineDiff = Integer.parseInt(o1.split(":")[1])
+                            - Integer.parseInt(o2.split(":")[1]);
+                    int columnDiff;
+                    try {
+                        columnDiff = Integer.parseInt(o1.split(":")[2])
+                                - Integer.parseInt(o2.split(":")[2]);
+                    }
+                    catch (NumberFormatException ex) {
+                        columnDiff = 1;
+                    }
+                    final int result;
+                    if (lineDiff != 0) {
+                        result = lineDiff;
+                    }
+                    else {
+                        result = columnDiff;
+                    }
+                    return result;
+                }
+            };
+            final List<String> actuals = lnr.lines().limit(expected.size())
```

Comment:
`.limit(expected.size())`
you should not limit actual errors. If there are more errors in `actuals` than a test must fail.

Reply:
done